omnibus/package-scripts/updater-deb/postinst

```diff
@@ -107,6 +111,9 @@ if [ -d "$CONFIG_DIR/runtime-security.d" ]; then
     chown -R root:root ${CONFIG_DIR}/runtime-security.d || true
 fi
 
+echo "api_key: deadbeef" > $CONFIG_DIR/datadog.yaml
```

Comment:
this is temporary until the updater passes through agent install script (with config file added at installation)